wiki/Beatmapping/Hitsound/en.md

```diff
@@ -21,13 +21,15 @@ Beatmappers can replace any default sound samples with custom samples using a [b
 
 ## Active hitsound
 
-An active hitsound correlates to a player's clicking by reaching its peak impact immediately when played. Drum sound samples are the most frequently used active hitsounds because they provide clear and immediate feedback.
+A hitsound is considered *active* if it is on a clickable hitobject edge, for example a circle or slider head. These help the player judge whether they clicked early or late by producing a clear impact sound at the time of clicking. If clicked correctly, the sound should match up with the beats in the song the objects are mapped to.
 
 ## Passive hitsound
 
-A passive hitsound does not correlate to a player's clicking. These are usually sound effects that accompany active hitsounds.
+If a hitsound is on a non-clickable edge, for example a slider repeat or spinner tail, it is considered *passive*. These always play at the same time as long as the player is interacting with the hitobject, unlike active hitsounds. Passive hitsounds neither need to have a clear impact nor be audible, but often are when over beats in the song to help the player maintain rhythm.
```

Comment:
```suggestion
If a hitsound is on a non-clickable part of a hit object, for example a slider repeat or spinner tail, it is considered *passive*. These always play at the same time as long as the player is interacting with the hit object, unlike active hitsounds. Passive hitsounds neither need to have a clear impact nor be audible, but often are when over beats in the song to help the player maintain rhythm.
```
also I've never heard people say "spinner tail". if im not just out of touch id pick a more common example, or reword that one

Reply:
spinner end would be good

Reply:
It is commonly used throughout the Ranking Criteria and is synonymous with "end", but I do think you have a point because its usage is a bit sporadic. Made [another PR to address that](https://github.com/ppy/osu-wiki/pull/6752).

Reply:
didn't expect my comment about "spinner tail" to turn into another PR, imo language about sliders is not related & doesn't matter cuz in different contexts people say different things

- "head" usually preferred over "start" for gameplay
- "end" *always* used for gameplay (you drop slider "ends", not "tails")
- "start"/"end" used for parts of skinning
- "start" relatively uncommon in mapping but "head" "end" "tail" all used

and ofc that^ changes over time and with different groups of people. just my experience

but across everyone I've talked to on osu I'm pretty sure I've never heard "spinner tail" which is why i think it should be "spinner end" or something. doesn't have to be consistent with slider language cuz they arent the same.

anyway, it's very minor detail, if u think it's fine as-is then w/e